bears/ember/EmberTemplateLintBear.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='ember-template-lint',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+):(?P<column>\d+)(?P<message>.*)'
+        )
+class EmberTemplateLintBear:
+    """
+    ember-template-lint lints templates and return error results.
+
+    This is commonly used through ember-cli-template-lint which
+    adds failing lint tests for consuming ember-cli applications.
+
+    https://github.com/ember-template-lint/ember-template-lint
+    """
+
+    LANGUAGES = {'Ember'}
```

Comment:
We can have a new language definition (not in this PR though) if Ember (template) is a different language from existing ones.

Reply:
Ember is a framework of JavaScript. It is doubtful where to put it as Ember (template) only handles the template part. So one option is to put it in JS but it is totally unrelated to linting that.

Reply:
It needs a new language definition in coala core.